### DISCUSSION TITLE
Server

### DISCUSSION THREAD

**Discussion by SilvaTechB** [community/community#123776] [Packages]
How do i change my repository server

**Comment by mishealkors** [comment]
Debian/Ubuntu (APT): Edit /etc/apt/sources.list with sudo nano /etc/apt/sources.list, replace URLs, then run sudo apt update.

CentOS/RHEL (YUM/DNF): Edit /etc/yum.repos.d/CentOS-Base.repo with sudo nano /etc/yum.repos.d/CentOS-Base.repo, change the base URL, then run sudo yum clean all and sudo yum update.

Arch Linux (Pacman): Edit /etc/pacman.d/mirrorlist with sudo nano /etc/pacman.d/mirrorlist, add/change to your new repository server, then run sudo pacman -Syy.

macOS (Homebrew): Change repository URL with cd "$(brew --repo)" and git remote set-url origin https://new.repo.url/Homebrew/brew.git, then run brew update.

Windows (Chocolatey): Add a new source with choco source add -n=NewRepo -s=https://new.repo.url/api/v2/, disable the old one with choco source disable -n=chocolatey, then run choco update.

**Reply by grandmas-favorite** [reply]
Thanks for sharing this info with the Community @mishealkors!
